components/nodemanager-service/pgdb/nodes_integration_test.go

```diff
@@ -261,26 +261,6 @@ func (suite *NodesIntegrationSuite) TestGetNodesCanFilterByProjects() {
 	suite.Equal("Taco Node", fetchedNodes[0].Name)
 }
 
-func (suite *NodesIntegrationSuite) TestGetNodesCanFilterByNoProjects() {
```

Comment:
We have not resolved what to do in the case where an empty list of project tags is requested. 
https://github.com/chef/automate/issues/55
Currently, if an empty list of projects is requested, the nodes will not be filtered by projects.

Reply:
That's how the IAM APIs work with project filtering. Is there a possibility this behavior will change?

Reply:
@bcmdarroch We have an issue #55 to look into how domain services should work if requests do not provide any projects. Are you saying that the current behavior that an empty list of project filters means don't filter projects or the "All" filter?

Reply:
Yeah the way we've implemented [ProjectsAuthorized](https://github.com/chef/automate/blob/master/components/authz-service/server/v2/authz.go#L110), when we get a request without projects, we check what projects the subject has permission on out of all of the possible projects and return only those allowed projects, to be passed to the domain.